models/food.js

```diff
@@ -16,5 +16,19 @@ module.exports = (sequelize, DataTypes) => {
       })
     })
   }
+
+  Food.destroyItem = function(id) {
+    return new Promise(function(resolve, reject) {
+      Food.findByPk(id)
```

Comment:
We might be able to reduce the number of queries by using Food.destroy, and just doing a where id is the id.

Reply:
I chacked to make sure, unfortunatly destroy() resolves even if no item was found with that Id. The spec wants us to return a 404 if that item doesnt exsist so im using the findByPk to handle that rejection.